src/test/frontend/handler/editor_test.cljs

```diff
@@ -43,22 +43,21 @@
       "Finds url in org link correctly"))
 
 (defn- set-marker
-  [marker content format new-marker]
+  [marker content format]
   (let [actual-content (atom nil)]
     (with-redefs [editor/save-block-if-changed! (fn [_ content]
                                                   (reset! actual-content content))]
-      (editor/set-marker {:block/marker marker :block/content content :block/format format}
-                         new-marker)
+      (editor/set-marker {:block/marker marker :block/content content :block/format format})
       @actual-content)))
 
 (deftest set-marker-org
-  (are [marker content new-marker expect] (= expect (set-marker marker content :org new-marker))
-    "TODO" "TODO content" "DOING" "DOING content"
-    "TODO" "## TODO content" "DOING" "## TODO content"
```

Comment:
"## TODO content" check seems to be necessary here, in case it wouldn't be toggled to "## DOING content" in Orgmode.

Reply:
Also in [113ae7e](https://github.com/logseq/logseq/pull/4661/commits/113ae7e687a942c4eb426c22c82c088768e6dcb6).